src/assets/20-tech-terms/20-beginner-tech-terms.md

```diff
@@ -0,0 +1,157 @@
+- `GET` method: The `GET` method is designed to request a specific resource.
+- `POST` method: The POST method is designed to submit data to be processed by a specified resource. It is akin to completing and sending a form.
+- `PUT` method: The `PUT` method is designed to replace an entire resource with a new content. It's pretty much similar to saving an edited document.
+- `DELETE` method: When the `DELETE` method is triggered, the server removes the resource identified by that request. This is like deleting a file.
+
+**Example scenario:**
+
+When you open any webpage, the browser sends a `GET` request to its server to get its page data. When you fill out a form it actually is sending your data over with a `POST` request.
+
+### 17. API
+An `API (Application Programming Interface)` is a set of specifications that allows different applications to communicate, and it provides an interface for developers to interact with other services. So we can picture it as a translator that helps two people speaking different languages understand each other.
+Key Features:
+- `Scalability`: Easy to extend and adapt for interoperation with other systems and applications
+- `Efficiency`: Can be embedded in other applications with great ease, thus giving you the freedom to utilize present functionality again instead of writing code from scratch.
+- `Security`: Ensures secure ways of accessing resources and a healthy trade of data.
+
+### 18. Cookies and Sessions
+`Cookies` and `sessions` are tools to store user data/state in our web applications. It enables websites to remember their own users so as to create a more personalized experience.
+
+**Cookies**:
+- **Meaning**: It is the tiny piece of data that is stored by a website on your browser.
+- **Goal**: For the purpose of remembering user activity, login details and preferences.
+- **Example**: Logging-in to certain websites, and checking the "remember me" box will store a cookie in your web browser.
+
+**Sessions**:
+- **Meaning**: Some type of temporary storage which includes server-side and client-side components that are used to store user data during the visit from the time a visitor sits on page until when they leave (session).
+- **Goal**: To maintain the state of the user and data during navigation across a website.
+- **Example**: When you add items to a crt on an e-commerce site, a session is created to keep the item in the cart until payment is finalized or the user leaves the site.
+
+## Authentication:
+
+### 19. SSO:
+`Single Sign-On,SSO` in short is an authentication process which allows users to enter one set of login credentials for multiple applications. It simplifies the user experience by removing the requirement to remember unique usernames and passwords for each application. That is nice for users but it also improves the security of your application and makes controlling user access easier by having one central location where validation occurs.
```

Comment:
At other instances throughout your text you have used this way of introducing an abbreviation

```suggestion
`Single Sign-On (SSO)` is an authentication process which allows users to enter one set of login credentials for multiple applications. It simplifies the user experience by removing the requirement to remember unique usernames and passwords for each application. That is nice for users but it also improves the security of your application and makes controlling user access easier by having one central location where validation occurs.
```

If you want to keep the expression as it is, there should be at last a space after the comma :)
```suggestion
`Single Sign-On, SSO` in short is an authentication process which allows users to enter one set of login credentials for multiple applications. It simplifies the user experience by removing the requirement to remember unique usernames and passwords for each application. That is nice for users but it also improves the security of your application and makes controlling user access easier by having one central location where validation occurs.
```